k-distribution/tests/regression-new/set-symbolic-tests/inset-11-spec.k.out

```diff
@@ -0,0 +1,19 @@
+  <k>
+    inSetResult ( Y:MyId in SET
```

Comment:
What is this testing?

Reply:
Same as above, this test was kept from the previous set of tests because it produces a correct answer, not because it tests any particular simplification.
If you want I can clean up the tests that do not apply any more. However, we already found [a bug](https://github.com/runtimeverification/haskell-backend/issues/3294) with one of those "useless" tests.